### ISSUE TITLE
REQUIREMENTS section is unclear on Rails version support

### ISSUE THREAD

**Issue by RepublicOfAppsLLC** [bwillis/versioncake#16]
Hi,

Cool gem, thanks!

The REQUIREMENTS section of the README is unclear to me when it says:

Rails >= v3.2 is supported in versioncake =< v1.1. Rails 4 support is included versioncake >= v1.2.

Does this mean that:
1. Rails 3.2 is only supported on versioncake <= v1.1 and that only Rails 4 is supported on versioncake >= v1.2?
2. Rails 3.2 is supported on versioncake v1.1 and v1.2 but Rails 4 is only supported on versioncake >= v1.2?

In other words, if I have a Rails 3.2 app, can I use versioncake > v1.1 (i.e. the latest version 1.3)?

There are still lots of Rails 3.2 apps out there and it would be nice to still use the latest version of versioncake with them.

Thanks,

Joel
